packages/main/test/specs/ResponsivePopover.spec.js

```diff
@@ -62,4 +62,16 @@ describe("ResponsivePopover general interaction", () => {
 		await btnCloseWithAttr.click();
 		assert.notOk(await popover.isDisplayedInViewport(), "Popover is closed.");
 	});
+
+	it("tests opening a popover from a responsive popover", async () => {
+		const btnOpenRP = await browser.$("#btnRpWithPopover");
+
+		await btnOpenRP.click();
+
+		const btnOpenPopover = await browser.$("#btnRpWithPopoverOpener");
+		await btnOpenPopover.click();
+
+		const popover = await browser.$("#btnRpWithPopoverInnerPopover");
+		assert.ok(await popover.isDisplayedInViewport(), "Popover is opened.");
```

Comment:
The test is relevant only for a mobile device, but I couldn't make emulating a mobile device to work with the test framework.